docs/source/user_guide/release_notes.md

```diff
@@ -1,3 +1,61 @@
+# Release Notes
+## v0.13.0rc1 - 2025.12.25
+This is the first release candidate of v0.13.0 for vLLM Ascend. We landed lots of bug fix, performance improvement and feature support in this release. Any feedback is welcome to help us to improve vLLM Ascend. Please follow the [official doc](https://vllm-ascend.readthedocs.io/en/latest) to get started.
+
+### Highlights
+- Improved the performance of DeepSeek V3.2. [#3995](https://github.com/vllm-project/vllm-ascend/pull/3995)
+- Improved the performance of Qwen3-Next. [#4595](https://github.com/vllm-project/vllm-ascend/pull/4595), [#4818](https://github.com/vllm-project/vllm-ascend/pull/4818), [#5322](https://github.com/vllm-project/vllm-ascend/pull/5322)
+- Qwen3-Next MTP with chunked prefill is supported now. [#4770](https://github.com/vllm-project/vllm-ascend/pull/4770)
+- [Experimental] Prefill Context Parallel and Decode Context Parallel are supported, but notice that it is an experimental feature now, welcome any feedback.
+
+### Other
+- OOM error on VL models is fixed now. [#5136](https://github.com/vllm-project/vllm-ascend/pull/5136)
+- Fixed an accuracy bug of Qwen3-Next-MTP when batched inferring. [#4932](https://github.com/vllm-project/vllm-ascend/pull/4932)
+- A new quantization method W4A16 is supported now. [#4541](https://github.com/vllm-project/vllm-ascend/pull/4541)
+- Cross-machine Disaggregated Prefill is supported now. [#5008](https://github.com/vllm-project/vllm-ascend/pull/5008)
+- Add UCMConnector for KV Cache Offloading. [#4411](https://github.com/vllm-project/vllm-ascend/pull/4411)
+- Support async_scheduler and disable_padded_drafter_batch in eagle. [#4893](https://github.com/vllm-project/vllm-ascend/pull/4893)
+- Support pcp + mtp in full graph mode. [#4572](https://github.com/vllm-project/vllm-ascend/pull/4572)
+
+Some general performance improvement:
+- Add new pattern for AddRmsnormQuant with SP, which could only take effect in graph mode. [#5077](https://github.com/vllm-project/vllm-ascend/pull/5077)
+- Add async exponential while model executing. [#4501](https://github.com/vllm-project/vllm-ascend/pull/4501)
+
+### Parameters changes:
+- `VLLM_ASCEND_ENABLE_DENSE_OPTIMIZE` is removed and `VLLM_ASCEND_ENABLE_PREFETCH_MLP` is recommend to replace as they always be enabled together. [#5272](https://github.com/vllm-project/vllm-ascend/pull/5272)
+- `VLLM_ENABLE_FUSED_EXPERTS_ALLGATHER_EP` is dropped now. [#5270](https://github.com/vllm-project/vllm-ascend/pull/5270)
+- `chunked_prefill_for_mla` in `additional_config` is dropped now. [#5296](https://github.com/vllm-project/vllm-ascend/pull/5296)
```

Comment:
There is another user impacted change: NZ is disabled for float case. https://github.com/vllm-project/vllm-ascend/pull/4878